elastic/metadata.csv

```diff
@@ -21,6 +21,11 @@ elasticsearch.cache.filter.size,gauge,,byte,,The size of the filter cache.,0,ela
 elasticsearch.cluster_status,gauge,,,,"The elasticsearch cluster health as a number: red = 0, yellow = 1, green = 2",0,elasticsearch,cluster status
 elasticsearch.docs.count,gauge,,document,,The total number of documents in the cluster across all shards.,0,elasticsearch,docs count
 elasticsearch.docs.deleted,gauge,,document,,The total number of documents deleted from the cluster across all shards.,0,elasticsearch,docs deleted
+elasticsearch.disk.avail,gauge,,byte,,The free disk space available to Elasticsearch.,0,elasticsearch,disk avail
+elasticsearch.disk.indices,gauge,,byte,,The disk space used by the node's shards.,0,elasticsearch,disk indices
+elasticsearch.disk.percent,gauge,,byte,,The total percentage of disk space in use. Calculated as disk.used / disk.total.,0,elasticsearch,disk percent
```

Comment:
Should the unit here be percentage?

Reply:
Hey Ruth! 👋 From [Elastic documentation's ](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-allocation.html) it should be `byte` for `disk.avail` and  `disk.total` (I'm using `CAT_ALLOC_PATH = '/_cat/allocation?v=true&format=json&bytes=b'`, so everything is in `byte`). As for `disk.percent` I guess it's percent since `(47.3/100.7)*100 ~= 46`.

```
shards disk.indices disk.used disk.avail disk.total disk.percent host      ip        node
     1         260b    47.3gb     43.4gb    100.7gb           46 127.0.0.1 127.0.0.1 CSUXak2
```

I'm updating the file! Thank you!